rpm/asteroidsyncservice.spec

```diff
@@ -40,7 +40,6 @@ Support for AsteroidOS watches in SailfishOS.
 rm -rf %{buildroot}
 %qmake5_install
 
-echo %{_userunitdir}
```

Comment:
You can squash this change (and the next one) into the previous comments with an interactive rebase https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d that makes up for cleaner history and is also generally a good practice for bissectability

Reply:
Debug line removed - squashing still to be done.